extensions/gdx-setup/src/com/badlogic/gdx/setup/data/extensions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
+    <extension>
+       <name>Overlap2D</name>
+       <description>level and ui editor runtime</description>
+       <package>com.underwaterapps.overlap2druntime</package>
+       <version>0.0.8</version>
+       <dependencies>
+           <core>com.underwaterapps.overlap2druntime:overlap2d-runtime-libgdx</core>
+       </dependencies>
+    </extension>
+</extensions>
```

Comment:
I think that each dependency should also have version of LibGDX that they are compatible with. For example my extension that relies heavily on scene2d.ui may not work with different version of LibGDX that I made and tested it with. 

Please look [here](https://github.com/kotcrab/VisEditor/wiki/VisUI#libgdx-compatibility).  I think that adding similar thing to xml file would be pretty useful, maybe it also would be good to mark old versions as outdated in gdx-setup.